pymc3/distributions/continuous.py

```diff
@@ -3902,7 +3882,7 @@ def logp(self, value):
         -------
         TensorVariable
         """
-        return at.log(self.interp_op(value) / self.Z)
+        return at.log(self.interp_op(value) / Z)
```

Comment:
A bit confusingly, we are not using `self` instances, the logp has to work as a standard function dependent only on it's inputs.